config/locales/en.yml

```diff
@@ -45,6 +46,8 @@ en:
   project_expected_start_date: 'Expected Start Date'
   project_expected_end_date: 'Expected End Date'
   project_user_name: 'User Name'
+  project_department_name: 'Name'
```

Comment:
make a generic name label and use when requires. same thing is being used in team_name